tests/infrastructure/golden_images/update_boot_source/conftest.py

```
@@ -123,3 +142,142 @@ def latest_rhel_release_versions_dict():
+    )
+
+
+@pytest.fixture()
+def created_persistent_volume_claim(unprivileged_client, data_import_cron_namespace):
+    def _get_pvc():
+        return list(
+            PersistentVolumeClaim.get(
+                dyn_client=unprivileged_client,
+                namespace=data_import_cron_namespace.name,
+            )
+        )[0]
```

Ruff (0.14.8)

Prefer `next(...)` over single element slice

Replace with `next(...)`

(RUF015)